Jenkinsfile.release

```diff
@@ -9,12 +9,14 @@ void tryAddKnownHost(String hostUrl){
 
 pipeline {
     agent {
-        docker { image 'pegasyseng/pantheon-build:0.0.5-jdk8' }
+        docker { image 'pegasyseng/pantheon-build:0.0.5-jdk11' }
     }
     parameters {
         string(name: 'BRANCH_NAME', defaultValue: 'master', description: '[MANDATORY] The name of the branch to create the release from')
-        string(name: 'RELEASE_VERSION', defaultValue: '', description: '[OPTIONAL] When empty: defaults to the current project version')
+        string(name: 'RELEASE_VERSION', defaultValue: '1.x', description: '[MANDATORY] The version you are deploying')
```

Comment:
`1.x` as the default seems misleading here - our versions would have three parts (1.x.x) and having a default will make it harder to detect if someone misses filling in this field.

Reply:
updated to x.y.z